spec/decorators/case/ico/sar_decorator_spec.rb

```diff
@@ -9,7 +9,7 @@
 
   describe '#type_printer' do
     it 'pretty prints Case' do
-      expect(ico_sar_case.decorate.pretty_type).to eq 'ICO appeal for SAR case'
+      expect(ico_sar_case.decorate.pretty_type).to eq 'ICO appeal(SAR)'
```

Comment:
Another place where we should have a space in front of `(SAR)` (and maybe capitalise `Appeal`).